src/multirust/config.rs

```diff
@@ -402,4 +410,51 @@ impl Cfg {
             Ok(name.to_owned())
         }
     }
+
+    pub fn set_telemetry(&self, telemetry_enabled: bool) -> Result<()> {
+        match telemetry_enabled {
+            true => self.enable_telemetry(),
+            false => self.disable_telemetry(),
+        }
+    }
+
+    fn enable_telemetry(&self) -> Result<()> {
+        let work_file = try!(self.temp_cfg.new_file());
+        
+        let _ = utils::ensure_dir_exists("telemetry", &self.multirust_dir.join("telemetry"), ntfy!(&NotifyHandler::none()));
+
+        try!(utils::write_file("temp", &work_file, ""));
+
+        try!(utils::rename_file("telemetry", &*work_file, &self.multirust_dir.join("telemetry_on")));
```

Comment:
Just personal preference: can you call this "telemetry-on" instead of "telemetry_on"?